Cython/Compiler/Nodes.py

```diff
@@ -10099,6 +10122,11 @@ def generate_loop(self, code, fmt_dict):
             reduction_codepoint = self.parent.privatization_insertion_point
         else:
             code.put("#pragma omp parallel")
+
+            # Add if-statements to code
```

Comment:
This seems a useless (and misleading) comment. There is only one such statement, and it is _obviously_ added to the code here.
```suggestion
```